### PR TITLE
Disable/deprecate the Manual Request Editor

### PR DESCRIPTION
**Pull request by thc202**
Deprecate all related classes and remove the extension from core
functionality, it will be provided by the requester add-on.
Address deprecation warnings in other classes.

---
WIP pending changes in the add-on.

### CODE REVIEW

File: zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java [hunk 1]
```diff
@@ -71,7 +72,12 @@
 import org.zaproxy.zap.utils.DisplayUtils;
 import org.zaproxy.zap.view.ZapMenuItem;
 
-/** Send custom crafted messages via HTTP or other TCP based protocols. */
+/**
+ * Send custom crafted messages via HTTP or other TCP based protocols
+ *
+ * @deprecated (2.12.0) Replaced by Requester add-on..
```
Comment (kingthorin) [inline zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java:78]: Just for consistency drop a period

Comment (kingthorin) [reply zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java:78]: No problem

Comment (thc202) [reply zap/src/main/java/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java]: Done.

**Review by kingthorin — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

**Comment by thc202** [comment]
Addressed a conflict.

**Pull request merged**